src/kernels/Conv_Winograd_v21_1_2_metadata.inc

```diff
@@ -71,28 +71,33 @@ amdhsa.kernels:
     - { .size: 4, .offset:  84, .value_kind: by_value, .value_type: i32, .name: out_w }
     - { .size: 8, .offset:  88, .value_kind: global_buffer, .value_type: f32, .name: bias_addr,    .address_space: global, .is_const: true }
```

Comment:
BTW, why only `bias_addr` is marked as `.is_const: true`
Let's make `in` and `weights` const also